package.json

```diff
@@ -27,12 +27,14 @@
   },
   "homepage": "https://github.com/javascriptdata/scikit.js#readme",
   "devDependencies": {
+    "@commitlint/cli": "^15.0.0",
+    "@commitlint/config-conventional": "^15.0.0",
     "@rollup/plugin-commonjs": "^21.0.1",
     "@rollup/plugin-node-resolve": "^13.0.6",
     "@rollup/plugin-typescript": "^8.3.0",
     "eslint": "^8.2.0",
+    "husky": "^7.0.4",
     "rollup-plugin-terser": "^7.0.2",
-    "typedoc": "^0.22.9",
-    "yarn": "^1.22.17"
+    "typedoc": "^0.22.9"
   }
```

Comment:
Looks great. Glad you removed yarn from the package.json. We should also switch the package.json's in scikitjs-browser and scikitjs-node right? The more I think about it, we should prob just switch everything over to npm and standardize on that.

Reply:
Definitely, I wanted to just remove it. Step. By step and make sure the current CI wasn’t breaking too much

Reply:
Sounds good. I'm happy with this PR then. Merge whenever you'd like. Great work.